src/libs/actions/User.ts

```diff
@@ -550,8 +551,13 @@ function playSoundForMessageType(pushJSON: OnyxServerUpdate[]) {
                         return playSound(SOUNDS.SUCCESS);
                     }
 
-                    // plain message
-                    if ('html' in message) {
+                    if (
+                        // plain message
+                        'html' in message &&
+                        // exclude mobile platforms, since they play sound when notification is delivered (in native code)
+                        Platform.OS !== 'android' &&
+                        Platform.OS !== 'ios'
```

Comment:
We'll need to remove the platform-specific condition. Typically we'll have a native variant of the file

Reply:
@Julesssss yeah, I've seen that you prefer file-based approach rather `if` conditions. But for this concrete case it doesn't make sense to replicate full file, right? We need to create platform-specific function `playReceiveSound`, correct?

Reply:
The rule applies app-wide, so we'd still need a way of doing it this way.

Reply:
Resolved it 👍